lib/system.nim

```diff
@@ -551,6 +551,10 @@ when notJSnotNims and not defined(nimSeqsV2):
 when notJSnotNims:
   include "system/hti"
 
+  proc writeToStdErr(msg: cstring, length: int)
+  proc writeToStdErr(msg: cstring)
```

Comment:
how about:
```nim
  proc writeToStdErr(msg: cstring) =
    writeToStdErr(msg, writeToStdErr.len)
```

that way, no need to define `proc writeToStdErr(msg: cstring)` on each platform